gm4_audere_shamir/data/gm4_audere_shamir/function/active_tool.mcfunction

```diff
@@ -4,18 +4,21 @@
 
 data modify storage gm4_audere_shamir:temp/player/tool SelectedItem set from entity @s SelectedItem
 execute store result score $tool_current_damage gm4_ml_data run data get storage gm4_audere_shamir:temp/player/tool SelectedItem.components."minecraft:damage"
-execute store result score $tool_max_damage gm4_ml_data run data get storage gm4_audere_shamir:temp/player/tool SelectedItem.components."minecraft:custom_data".gm4_audere_shamir.max_durability
+execute store result score $tool_max_damage gm4_ml_data run data get storage gm4_audere_shamir:temp/player/tool SelectedItem.components."minecraft:custom_data".MaxDurability
+# | MaxDamage is set by gm4_moneo_shamir:store_maximum
```

Comment:
Does this mean Audere is now reliant on Moneo? While that's technically fine while Moneo is in base Metallurgy it might cause problems if Moneo is ever updated / removed since it's unlikely this would be thought of then